src/clientdynamicinfo.h

```diff
@@ -35,17 +40,36 @@ struct ClientDynamicInfo
 				abs(real_hud_scaling - other.real_hud_scaling) < 0.001f;
 	}
 
-	static v2f32 calculateMaxFSSize(v2u32 render_target_size) {
+#ifndef SERVER
+	static const ClientDynamicInfo getCurrent() {
```

Comment:
Why is the return type const?

Reply:
I don't know. I've now undone that change.